Cloud_Providers.md

```diff
@@ -0,0 +1,50 @@
+
+## Amazon Elastic Kubernetes Service (EKS)
+
+### Deploying SAS Viya Logging and Monitoring on Amazon EKS
+
+In order to use the SAS Viya monitoring and logging components in an Amazon EKS environment, you must deploy monitoring and logging using ingress. See the [Ingress sample](/samples/ingress/README.md) for deployment information.
+
+No other customization is needed to deploy and run the SAS Viya logging and monitoring components on Amazon EKS.
+
+### Using Amazon EKS Logging and Monitoring Applications With SAS Viya
+
+If you deploy SAS Viya in an Amazon EKS environment, you can view metric information by using either the SAS Viya monitoring components or Amazon's monitoring application, CloudWatch. In order to use CloudWatch to monitor SAS Viya metrics, you must deploy the CloudWatch agent for Prometheus, make configuration changes to convert the Prometheus metrics collected from SAS Viya into performance log events, and map the log events to CloudWatch metrics. Follow the steps in the [Amazon CloudWatch Integration sample](/samples/cloudwatch/README.md) to use CloudWatch to monitor a selected group of SAS Viya metrics. Not all SAS Viya metrics are supported by this sample, because supporting all SAS Viya metrics would require every possible combination of labels from every SAS Viya source and every metric to be mapped to CloudWatch metric types. This sample might be updated in later releases to include additional combinations of dimensions for key metrics.
+
+Amazon CloudWatch can also display log messages collected from Fluent Bit, which is the application used by the SAS Viya logging solution. However, current samples do not provide the configuration needed to integrate SAS Viya log processing with CloudWatch.  
+
+## Google Kubernetes Engine (GKE)
+
+### Deploying SAS Viya Logging and Monitoring on GKE
+
+In order to use the SAS Viya monitoring and logging components in an GKE environment, you must deploy monitoring and logging using ingress. See the [Ingress sample](/samples/ingress/README.md) for deployment information.
+
+Depending on the size of your cluster, you might need to increase the default size of the virtual machines in your nodepool and to split the workload among multiple nodes in order for the logging and monitoring components to deploy and run successfully. 
+
+### Using GKE Logging and Monitoring Applications With SAS Viya
+
+Google provides support for logging and monitoring through the Google Operation Suite application. Currently, no samples are available to integrate SAS Viya metrics and log messages with Google Operations Suite. 
+
```

Comment:
Can we change the slant of this to be more positive: _In the future, samples may be developed showing how SAS Viya metrics and log monitoring can be integrated with Google Operations Suite._

Reply:
Sentence changed to "In future releases, samples might be developed to provide procedures for integrating SAS Viya metrics and log monitoring with the applications in Google Cloud's operations suite."